scripts/o3de/o3de/download.py

```diff
@@ -363,6 +405,9 @@ def add_parser_args(parser):
     parser.add_argument('-f', '--force', action='store_true', required=False,
                             default=False,
                             help = 'Force overwrite the current object')
+    parser.add_argument('-s', '--use-source-control', action='store_true', required=False,
+                            default=False,
+                            help = 'Acquire from source control instead of downloading a .zip archive.  Requires that the object has ')
```

Comment:
Seems strange that the option is `-s`, but it doesn't start with the letter of s.

How about `-usc` instead?
Also it would better for showing the usage if the long option appeared first, so that the usage text outputs `--use-source-control`
```suggestion
    parser.add_argument('--use-source-control', '-s', action='store_true', required=False,
                            default=False,
                            help = 'Acquire from source control instead of downloading a .zip archive.  Requires that the object has ')
```